Makefile

```diff
@@ -0,0 +1,33 @@
+LDFLAGS = -ldflags "-X main.gitSHA=$(shell git rev-parse HEAD)"
+
+OS := $(shell uname)
+
+build: clean
+	go build $(LDFLAGS) -o align
+
+test:
+	go test -v
+
+clean:
+	go clean
+	rm -f align
+	rm -f bin/*
```

Comment:
Remove this line if no binary is being placed in a `bin` directory or just update the build target to put the binary in a bin directory and include that bin directory in the .gitignore file.